src/test-run/index.js

```diff
@@ -43,7 +43,8 @@ export default class TestRun extends Session {
         this.injectable.scripts.push('/testcafe-driver.js');
         this.injectable.styles.push('/testcafe-ui-styles.css');
 
-        this.errs = [];
+        this.errs                = [];
+        this.driverResponseCache = {};
```

Comment:
```js
this.lastDriverResponse  = { statusId, response };
```

Let's split it into two vars to avoid confusion like we have we drive command:
```js
this.lastDriverStatusId = driverStatus.id;
this.lastDriverStatusResponse = response;
```